### ISSUE TITLE
how to speed up to model pruned?

### ISSUE THREAD

**Issue by eeric** [jacobgil/pytorch-pruning#11]
the speed of model pruned did not improve, how to your work?

**Comment by eeric** [comment]
The ResNet model pruned impletement faster little about 10%, code in
https://github.com/eeric/channel_prune

the reason that non-tensor layers (e.g., batch normalization
and pooling layers) took up more than 40% of the inference
time on GPU.

**Comment by guoxiaolu** [comment]
How to get this "the reason that non-tensor layers (e.g., batch normalization
and pooling layers) took up more than 40% of the inference
time on GPU." Are there some papers or other things?
@eeric

**Comment by eeric** [comment]
test self

**Comment by eeric** [comment]
@buaaJeremyduan, thanks!

**Comment by kssk16** [comment]
@Kuldeep-Attri did you find out the reason? 
My observation in that regard is same. Let me know if you have any leads?
